server/public/model/client4.go

```diff
@@ -7002,8 +7002,8 @@ func (c *Client4) GetJob(ctx context.Context, id string) (*Job, *Response, error
 }
 
 // GetJobs gets all jobs, sorted with the job that was created most recently first.
-func (c *Client4) GetJobs(ctx context.Context, page int, perPage int) ([]*Job, *Response, error) {
-	r, err := c.DoAPIGet(ctx, c.jobsRoute()+fmt.Sprintf("?page=%v&per_page=%v", page, perPage), "")
+func (c *Client4) GetJobs(ctx context.Context, page int, perPage int, jobType string, status string) ([]*Job, *Response, error) {
```

Comment:
Any problems with changing this signature?

Reply:
I don't think so.